ref_impl/package.json

```diff
@@ -11,14 +11,15 @@
         "url": "https://github.com/Microsoft/BosqueLanguage"
     },
     "dependencies": {
-        "@types/node": "~10.12.18"
+        "@types/node": "~10.12.18",
+        "typescript": "^3.4.3"
```

Comment:
You should add it to `devDependencies` instead of `dependencies`

Reply:
Hi @Hongarc. Thanks for the review.

I think if I put it in `devDependencies` instead, people won't be able to build the software in `ref_impl/bin/` which contains the interpreter, compiler to build/run their code if they do just run install without dev dependencies install switch.

What do you think?

Reply:
> people won't be able to build the software builds
If they are the user, they don't need to build `bin` file.
We publish `bin` file in the release.

Reply:
If that is the case, then sure. Thanks.